config/locales/en/activemodel.yml

```diff
@@ -71,6 +71,12 @@ en:
             postcode:
               blank: Enter a postcode
               invalid: Enter a postcode in the right format
+        aggregated_cash_income:
+          credits:
+            attributes:
+              none_selected:
+                blank: Select if you make these payments in cash
+                others_present: Select payments in cash or None of the above
```

Comment:
These errors > blank and others_present say payments

aren't these income so it should be received ?